creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -101,6 +101,13 @@ class SnapbackSM {
     // Toggle to switch logs
     this.debug = true
 
+    // Start at a random userId to avoid biased processing of early users
+    this.lastProcessedUserId = Utils.randomIntFromIntervalInclusive(
+      0,
+      10_000_000
```

Comment:
i guess this is fine since if total num users is eg 1mil, and this is initted to 5mil, then the first run will no-op and immediately cycle back to 0

we can just call https://discoveryprovider.audius.co/latest/user and use that as max to prevent a wasteful first run

i'm glad you randomized starting point tho 👍